src/components/molecules/PrevNextBar.js

```diff
@@ -24,9 +25,9 @@ export function PrevNextBar(props) {
   }
 
   return (
-    <div id={id} className="flex p-2 inline-flex w-full">
+    <div id={id} className="flex p-2 bg-gray-300 justify-end">
       <Router>
```

Comment:
as in standup :)

Reply:
created a PrevNext component instead . Thanks

Reply:
Can you delete this component if its not being used